docs/docs/20-usage/71-project-settings.md

```diff
@@ -16,6 +16,15 @@ Your Version-Control-System will notify Woodpecker about events via webhooks. If
 
 Enables handling webhook's pull request event. If disabled, then pipeline won't run for pull requests.
 
+## Allow deployments
+
+Allows you to start a pipeline with `deploy` event from a succeeded pipeline.
+
+:::danger
+Only enable this option if you trust all users that have push access to your repository.
+Otherwise these users are able to steal secrets only available to `deploy` events.
```

Comment:
```suggestion
Only activate this option if you trust all users who have push access to your repository.
Otherwise, these users will be able to steal secrets that are only available for `deploy` events.
```